### PR TITLE
[REMPL_APOLLO_DEVTOOLS] apollo-inspector added

### PR DESCRIPTION
**Pull request by vejrj**

### CODE REVIEW

File: packages/rempl-apollo-devtools/src/publisher/publishers/graphiql-publisher.ts [hunk 1]
```diff
@@ -23,7 +22,7 @@ export class GraphiQLPublisher {
   private attachMethodsToPublisher() {
     this.apolloPublisher.provide(
       "graphiql",
-      (activeClientId: string, graphQLParams: FetcherParams) => {
+      (activeClientId: string, graphQLParams: any) => {
```
Comment (alloy) [inline packages/rempl-apollo-devtools/src/publisher/publishers/graphiql-publisher.ts:25]: Why?

File: packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/apollo-operations-tracker-utils.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  const definition =
+    query && query.definitions && query.definitions.length > 0
+      ? (query.definitions[0] as OperationDefinitionNode)
+      : null;
+  const operationName = definition ? definition.name?.value : "name_not_found";
+
+  return operationName;
+};
+
+export const isNumber = (input: string | number | undefined = "NA") => {
```
Comment (alloy) [inline packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/apollo-operations-tracker-utils.ts:13]: Tip: make this into a TS type-guard next time, so you get benefits of type narrowing at the call-site.

File: packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/utils/apollo-operations-tracker-utils.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  return true;
+};
+
+export const copyToClipboard = async (obj: unknown) => {
+  try {
+    await window.navigator.clipboard.writeText(JSON.stringify(obj));
+  } catch (error) {
+    console.log(`failed to copy`, error);
+  }
+};
```
Comment (alloy) [inline packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/utils/apollo-operations-tracker-utils.ts:28]: Same file again? What gives?

**Review by alloy — APPROVED** [review]
Great stuff! Only minor feedback that doesn't need to block deployment.

File: packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/apollo-operations-tracker-utils.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+export const getOperationName = (query: DocumentNode) => {
+  const definition =
+    query && query.definitions && query.definitions.length > 0
+      ? (query.definitions[0] as OperationDefinitionNode)
+      : null;
+  const operationName = definition ? definition.name?.value : "name_not_found";
+
+  return operationName;
+};
```
[the remark this answers is not in the document]
Comment (jpsahoo14) [reply packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/apollo-operations-tracker-utils.ts:11]: Will add this change in the next iteration.

**Pull request merged**